packages/api-server/README.md

```diff
@@ -0,0 +1,19 @@
+# Setup
+
+Let's create a virtual environment named `venv` for all this Python magic:
+```
+sudo apt install python3-venv
+python3 -m venv venv
+```
+
+Now let's activate the Python virtual environment so we can install all sorts of things into it without breaking our system:
+```
+. venv/bin/activate
+pip install flask python-dotenv
+```
```

Comment:
QQ: Have you tried `Poetry` to handle a python project? 

https://python-poetry.org/docs/

Reply:
I mostly copied the scaffolding from @codebot 's experiment. That looks pretty cool, we will have to see if it works well in the ros2 ecosystem.